core/refresh.ts

```diff
@@ -110,7 +110,7 @@ export async function refreshPosts(): Promise<void> {
             filters = prepareFilters(await loadFilters({ feedId: feed.id }))
           }
           for (let origin of posts) {
-            if (feedStore.deleted || wasAlreadyAdded(feed, origin)) {
+            if (feedStore.deleted) {
```

Comment:
Why did we make this change?

Reply:
Accidentally. I'm doing this for testing